src/gui/clips/AutomationClipView.cpp

```diff
@@ -486,5 +486,57 @@ void AutomationClipView::scaleTimemapToFit( float oldMin, float oldMax )
 	m_clip->generateTangents();
 }
 
+//! Split this Clip.
+/*! \param pos the position of the split, relative to the start of the clip */
+bool AutomationClipView::splitClip( const TimePos pos )
+{
+	setMarkerEnabled( false );
```

Comment:
I fo understand that you copy pasted, it would be good if you do the style changes according to the guidelines tho.